webrender_traits/src/types.rs

```diff
@@ -191,6 +191,53 @@ pub struct ColorF {
 }
 known_heap_size!(0, ColorF);
 
+#[derive(Clone, Copy, Hash, Eq, Debug, Deserialize, PartialEq, PartialOrd, Ord, Serialize)]
+pub struct ColorU {
```

Comment:
It feels slightly wrong that both `ColorU` and `ColorF` are exposed in webrender_traits. Shouldn't one of them be just an implementation detail (hidden inside `webrender`)?